examples/poloniex.go

```diff
@@ -51,10 +48,10 @@ func main() {
 		}
 	*/
 
-	// Get Candle ( OHLCV )
+	// Get CandleStick chart data ( OHLCV )
 	/*
-		markets, err := poloniex.GetHisCandles("BTC-LTC", "hour")
-		fmt.Println(markets, err)
+		candles, err := poloniex.ChartData("BTC_XMR", 300, time.Now().Add(-time.Hour), time.Now())
+		fmt.Println(candles, err)
 	*/
```

Comment:
can you make a loop for the results that prints something pretty?

similar to the "Market history" example.